src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -530,10 +550,12 @@ int32_t AppleCryptoNative_SslSetEnabledCipherSuites(SSLContextRef sslContext, co
         {
             cipherSuites16[i] = (SSLCipherSuite)cipherSuites[i];
         }
+
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
         OSStatus status = SSLSetEnabledCiphers(sslContext, cipherSuites16, (size_t)numCipherSuites);
 #pragma clang diagnostic pop
+
```

Comment:
since this file doesn't have any changes please revert

Reply:
This is change related to PR #39984, ported from 3.0 https://github.com/dotnet/corefx/pull/39984#discussion_r310855086

Reply:
It's putting back the whitespace that a previous change removed, and making the master and 3.0 copy be the same again.  Generally not a fan, but technically correct (and desirable) in this case.